docker/Dockerfile.ubuntu

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:latest
+
+EXPOSE 9090
+
+RUN apt-get update -qq && apt-get install -y -qq --no-install-recommends \
+       build-essential cmake libboost-all-dev
+
+RUN mkdir -p /opt/confluo
+COPY . /opt/confluo
+
+WORKDIR /opt/confluo
+RUN mkdir build \
+    && cd build \
+    && cmake -DBUILD_TESTS=OFF -DWITH_PY_CLIENT=OFF -DWITH_JAVA_CLIENT=OFF .. \
+    && make -j8 install
+
+RUN mkdir -p /var/db
+VOLUME /var/db
+
+ENTRYPOINT ["confluod"]
+CMD ["--address=0.0.0.0", "--port=9090", "--data-path=/var/db", "2>/var/log/confluo.stderr 1>/var/log/confluo.stdout"]
```

Comment:
I don't think this will work correctly for logging. Perhaps try:

ENTRYPOINT["sh", "-c", "confluod --address=0.0.0.0 --port=9090 --data-path=/var/db --data-path=/var/db"]

Reply:
Simplified entrypoint - do not need any redirection of sterr/stdout as it works out of the box